waves_gateway/service/coin_transaction_consumer_impl.py

```diff
@@ -119,15 +124,39 @@ def _handle_transaction_receiver(self,
 
         # ---------- Pre-Check ----------
 
+        sender_array = []
+        if senders:
```

Comment:
senders is not a boolean

Reply:
But when the object is filled, it will jump into this statement, or did I misunderstood that?

Reply:
I am Not Sure. In Any Case, please only use boolean expressions.